owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -282,10 +290,38 @@ class PassCodeActivity : AppCompatActivity() {
         }
     }
 
+    private fun lockScreen() {
+        val timeToUnlock = passCodeViewModel.getTimeToUnlockLeft()
+        if (timeToUnlock > 0) {
+            binding.lockTime.isVisible = true
+            for (editText: EditText? in passCodeEditTexts) {
+                editText?.isEnabled = false
+            }
+            object : CountDownTimer(timeToUnlock, 1000) {
+                override fun onTick(millisUntilFinished: Long) {
+                    val minutes = ((millisUntilFinished + 1000) / 1000) / 60
+                    val seconds = ((millisUntilFinished + 1000) / 1000) % 60
+                    val timeString = String.format("%02d:%02d", minutes, seconds)
+                    binding.lockTime.text = getString(R.string.lock_time_try_again, timeString)
+                }
+
+                override fun onFinish() {
+                    binding.lockTime.isVisible = false
+                    for (editText: EditText? in passCodeEditTexts) {
+                        editText?.isEnabled = true
+                    }
+                    passCodeEditTexts[0]?.requestFocus()
```

Comment:
Try to use `passCodeEditTexts.first()` instead of `passCodeEditTexts[0]`  It's more Kotlin friendly!

Reply:
I didn't know about this function, thanks! I'll apply it from now on.